src/megachatapi_impl.cpp

```diff
@@ -7907,7 +7909,12 @@ void MegaChatCallHandler::onDestroy(rtcModule::TermCode reason, bool /*byPeer*/,
     {
         chatid = chatCall->getChatid();
         unique_ptr<MegaChatRoom> chatRoom(megaChatApi->getChatRoom(chatid));
-        assert(chatRoom);
+        if (!chatRoom)
+        {
+            // ChatRoom has been deleted => logout
```

Comment:
I would keep an `assert(false)` here, since it represents a bug on the cleanup phase.